webpack_config/webpack.prod.js

```diff
@@ -0,0 +1,43 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const FaviconsWebpackPlugin = require('favicons-webpack-plugin');
+const common = require('./webpack.common.js');
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+const path = require('path');
+
+const config = merge(common, {
+  output: {
+    publicPath: './'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$|\.css$/,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: [
+            {
+              loader: 'css-loader'
+            },
+            {
+              loader: 'resolve-url-loader'
+            },
+            {
+              loader: 'sass-loader'
+            }
+          ]
+        })
+      }
+    ]
+  },
+  plugins: [
+    new UglifyJSPlugin(),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('production')
+    }),
+    new FaviconsWebpackPlugin(path.join(__dirname, '../src/assets/imgs/favicon.png'))
```

Comment:
Unable to comment on the PNG itself, but it looks like it's just a blue circle (No white inside) and it's got a bunch of whitespace padding on it.